app/views/catalog/_show_default.html.erb

```diff
@@ -4,12 +4,17 @@
   This is pretty hackish for now - figure out a better way to do this into the CatalogController! -%>
 
 <%
-	@item = Source.find(@document.id.split(" ")[1]) # hackady-hack! Source 1234567 -> 1234567
+	if RISM::MAIN_BIBLIOGRAPHIC_RECORD_TYPE == "Source"
```

Comment:
Isn't it possible to convert `RISM::MAIN_BIBLIOGRAPHIC_RECORD_TYPE ` to model or class?

Reply:
I tried different approaches before publishing this one. This is the most concise approach I found, because this value can be use as such for sending values to Solr in app/controllers/catalog_controller.rb.  Otherwise, I should have to do a couple of conversions to string in `redirect_legacy_values`  and in `configure_blacklight`.  So I stepped back and tried with a string, and now it is much shorter.